streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractWindowBytesStoreTest.java

```diff
@@ -80,6 +80,15 @@
     static final long SEGMENT_INTERVAL = 60_000L;
     static final long RETENTION_PERIOD = 2 * SEGMENT_INTERVAL;
 
+    final long defaultStartTime = SEGMENT_INTERVAL - 4L;
+
+    final KeyValue<Windowed<Integer>, String> zero = windowedPair(0, "zero", defaultStartTime);
+    final KeyValue<Windowed<Integer>, String> one = windowedPair(1, "one", defaultStartTime + 1);
+    final KeyValue<Windowed<Integer>, String> two = windowedPair(2, "two", defaultStartTime + 2);
+    final KeyValue<Windowed<Integer>, String> three = windowedPair(3, "three", defaultStartTime + 2);
```

Comment:
Is this supposed to be defaultStartTime + **_3_**?

Reply:
Thanks for your comment, but here, I make it as `defaultStartTime + 2` on purpose, to test the case that when window starts time is the same, the forward/backward query API can return the order as expected. I updated the PR description to make it clear. Thank you.